documentation/src/main/asciidoc/topics/config_examples/rolling_upgrade_remote_store.xml

```diff
@@ -8,6 +8,14 @@
                  protocol-version="2.5"
                  raw-values="true"
                  segmented="false">
+      <!-- Configures authentication and encryption according to the security realm of the source cluster. -->
+      <security>
+        <authentication server-name="infinispan">
+          <digest username="admin"
+                  password="changeme"
+                  realm="default"/>
+        </authentication>
```

Comment:
@gustavonalle Hi, this updates the example remote store for rolling upgrade with authentication. It should be minor but could you take a look and give your opinion?